wasmer/wasmer.hh

```diff
@@ -1,17 +1,30 @@
 
 #if !defined(WASMER_H_MACROS)
+
```

Comment:
Please mention the changes to `wasmer.h` and `wasmer.hh` in the PR. E.g. "comments from upstream".

Reply:
Updated description.